src/lib/config.ts

```diff
@@ -1319,7 +1310,7 @@ export function unset(...target) {
 
     @hidden
  */
-export async function save(storage: "local" | "sync" = get("storageloc")) {
+export async function save(storage: "local" | "sync" = "local") {
```

Comment:
Why not completely remove the `storage` argument?

Reply:
I was being lazy when I threw together the proof of concept and I forgot to tidy it up :)